packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -184,14 +198,32 @@ const node = new VirtualNode(this, 'node', {
       idle: cdk.Duration.seconds(5),
     },
   })],
+  backendDefaultsClientPolicy: appmesh.ClientPolicy.fileTrust({
+    certificateChain: 'path-to-certificate',
+  }),
   accessLog: appmesh.AccessLog.fromFilePath('/dev/stdout'),
 });
 
+const virtualService = new appmesh.VirtualService(stack, 'service-1', {
+  virtualServiceName: 'service1.domain.local',
+  mesh,
+  clientPolicy: appmesh.ClientPolicy.fileTrust({
+    certificateChain: 'path-to-certificate',
+    ports: [8080, 8081],
+  }),
+});
+
+node.addBackend(virtualService);
```

Comment:
This example is exactly the same as above... what's the point of having the same identical example twice in the ReadMe?